src/parsing_astable.jl

```diff
@@ -0,0 +1,95 @@
+function conditionally_add_symbols!(inputs_to_function, lhs_assignments, col)
```

Comment:
How about making the signature more explicit? At least, `lhs_assignments::AbstractDict` would make things clearer. If you rely on the order, better even use `::OrderedCollections.OrderedDict`. Same below.

Reply:
added.